common/Timer.cc

```diff
@@ -36,9 +36,13 @@ FlowId Timer::getFlowEdge() {
 
 Timer::~Timer() {
     auto clock = chrono::steady_clock::now();
-    auto dur = std::chrono::duration<double, std::milli>(clock - start);
-    log.debug("{}: {}ms", this->name.str, dur.count());
-    sorbet::timingAdd(this->name, start, clock, move(args), self, prev);
+    auto dur = clock - start;
+    if (dur > std::chrono::microseconds(250)) {
```

Comment:
Yeah, this will throw off LSP metrics. I won't have any way to tell how responsive LSP really is. I'll have proxy metrics, like percentage of time taking slow path, but that's about it. I really want to know what the P50, P90, etc are for things like message latency as that's a clear indication of how long users have to wait for things like autocomplete.

Can we efficiently disable this logic with a `const bool` that specifies if the timer should opt-in to this optimization (or opt-out)?

Reply:
(For LSP case, I care less about overhead of timers for fast ops as messages are generally limited by human I/O speed in the interactive case.)

Reply:
Actually, this limit is so low I don't think we do anything that's that fast.